openui5/tsconfig.json

```diff
@@ -2,16 +2,10 @@
     "compilerOptions": {
         "module": "commonjs",
         "target": "es6",
-        "noImplicitAny": false,
+        "noImplicitAny": true,
         "strictNullChecks": false,
-        "types": [],
+        "types": ["jquery"],
```

Comment:
The `/// <reference path="../jquery/index.d.ts" />` line should still be removed.